src/features/chat/components/ChatMessage.tsx

```diff
@@ -0,0 +1,106 @@
+import InsertEmoticonIcon from "@mui/icons-material/InsertEmoticon";
+import MoreVertIcon from "@mui/icons-material/MoreVert";
+import ReplyIcon from "@mui/icons-material/Reply";
+import { Box, Typography, Avatar } from "@mui/material";
+import useTheme from "features/theme/useTheme";
+import { useState } from "react";
+
+type ChatMessageProps = {
+  isAuthUser: boolean;
+  content: string;
+  avatar: string;
```

Comment:
I would update this to profilePic so we don't have to transform the property name we get from the response